### PR TITLE
Service- Recycling- get layer information done

### PR DESCRIPTION
**Pull request by RamalingamThangavel**
Service- Recycling- get layer information done

### CODE REVIEW

File: src/main/groovy/org/sharegov/mdcgis/AddressController.groovy [hunk 1]
```diff
@@ -63,6 +63,54 @@ class AddressController {
 	MailSender mailSender
 
 	/**
+	 * get recycling layer information.
```
Comment (fiallega) [inline src/main/groovy/org/sharegov/mdcgis/AddressController.groovy]: this method is not about recycling, it is for any layer

File: src/main/groovy/org/sharegov/mdcgis/AddressController.groovy [hunk 2]
```diff
@@ -125,6 +161,31 @@ class AddressController {
 		json.call(answer)
 	}
 
+	/**
+	 * Client and get Municipality ID
```
Comment (fiallega) [inline src/main/groovy/org/sharegov/mdcgis/AddressController.groovy]: can you explain what this method does ? "Client an get Municipality ID" does not sound right

File: src/main/groovy/org/sharegov/mdcgis/AddressService.groovy [hunk 3]
```diff
@@ -44,6 +44,35 @@ class AddressService {
 	CacheManager cacheManager
 	GisConfig gisConfig
 
+	/**
+	 * get X and Y coordinates of address.
```
Comment (fiallega) [inline src/main/groovy/org/sharegov/mdcgis/AddressService.groovy]: you want to explain this better.
You want to explain that if more of one candidate you are returning a null

File: src/main/groovy/org/sharegov/mdcgis/AddressController.groovy (hunk 1 above)
Comment (RamalingamThangavel) [reply src/main/groovy/org/sharegov/mdcgis/AddressController.groovy]: corrected.

File: src/main/groovy/org/sharegov/mdcgis/AddressController.groovy (hunk 2 above)
Comment (RamalingamThangavel) [reply src/main/groovy/org/sharegov/mdcgis/AddressController.groovy]: The comments were wrong. Corrected the comments

File: src/main/groovy/org/sharegov/mdcgis/AddressService.groovy (hunk 3 above)
Comment (RamalingamThangavel) [reply src/main/groovy/org/sharegov/mdcgis/AddressService.groovy]: modified the comments

**Pull request merged**